composer.json

```diff
@@ -64,7 +61,7 @@
             "tools\/vendor\/bin\/php-cs-fixer fix --using-cache=no"
         ],
         "test" : [
-            "vendor\/bin\/phpunit"
+            "vendor\/bin\/phpunit --coverage-html var/phpunit/coverage/html --coverage-filter src"
```

Comment:
The coverage config has been inlined here and removed from the XML configuration since Infection does not recognize it properly.